nixos/modules/services/web-apps/nextcloud.nix

```diff
@@ -420,6 +428,7 @@ in {
               'dbtype' => '${c.dbtype}',
               'trusted_domains' => ${writePhpArrary ([ cfg.hostName ] ++ c.extraTrustedDomains)},
               'trusted_proxies' => ${writePhpArrary (c.trustedProxies)},
+              ${optionalString (c.extraConfig != null) c.extraConfig}
```

Comment:
It would make sense to leave an `example` in the option, otherwise a developer would have to read the module's code to know how to use this.